### ISSUE TITLE
On Mac Mini M2 Pro (2023) Power Consumption displays "denied" symbol

### ISSUE THREAD

**Issue by dirk68-fu** [exelban/stats#1347]
On an Mac Mini M2 Pro (2023) the Power Consumption Widget show the 🚫 symbol instead of a value.
The Sensor Dropdown menu shows correct values, but the widget does not. 
PS: Great App!

**Comment by exelban** [comment]
Because you have no enabled sensors.

**Comment by dirk68-fu** [comment]
Why can't the values from the dropdown menu be displayed in the menu bar? They seem to be correct.

**Comment by exelban** [comment]
because you need to enable them

**Comment by dirk68-fu** [comment]
I did enable one of them and it still shows 🚫.
I don't understand your answer: Is this not supported for my model or did I not enable it correctly.
Im pretty sure I switched at least one switch in and it still showed 🚫.

**Comment by exelban** [comment]
please share the screenshot of the sensors module settings

**Comment by dirk68-fu** [comment]
![image](https://user-images.githubusercontent.com/9881136/223935516-6f1f3317-521f-409a-96c6-e9967d54b176.png)

**Comment by dirk68-fu** [comment]
Thank you for your patience.
It works now, but I don't know why it didn't before. I had to remove the sensor widget and reenable it to make it work.
